.github/CC_SERVER_INFO.md

```diff
@@ -0,0 +1,45 @@
+<!--Thank you for your report. Please click submit new issue below.-->
+<!-----------Remember to tick all relevant boxes when done!---------->
+
+<!------------------------------------------------------------------->
+<!------------------------------------------------------------------->
+<!------------------ DO NOT MODIFY THE TEXT BELOW ------------------->
+<!------------------------------------------------------------------->
+<!------------------------------------------------------------------->
+
+
+##### AC HASH/COMMIT:
+
+https://github.com/chromiecraft/azerothcore-wotlk/commit/7c21d567d59153b37978756a5bf02daf6998e665
+
+##### OPERATING SYSTEM:
+
+Ubuntu 20.04
+
+##### MODULES:
+
+- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
+- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
+- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
+- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
+- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
+- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
+- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
+- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
+- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
+- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
+- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
+- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
+- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
+- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
+- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
+- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
+- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

```

Comment:
While we are editing it, why don't use this?

<details>
  <summary>Modules</summary>

 - module 1
 - module 2  
 
</details>

Reply:
not sure on this one. What do you think @FrancescoBorzi ?

Reply:
I guess it's fine the way it is